Library/Homebrew/formula.rb

```diff
@@ -818,8 +850,10 @@ def post_install_defined?
 
   # @private
   def run_post_install
-    build, self.build = self.build, Tab.for_formula(self)
-    post_install
+    with_logging(:post_install) do
+      build, self.build = self.build, Tab.for_formula(self)
```

Comment:
It looks weird to have this line inside block, especially considering the ensure block